Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -195,5 +195,29 @@ public static void CreatePortal(Session session, params string[] parameters)
         {
             LandblockManager.AddObject(PortalObjectFactory.CreatePortal(1234, session.Player.Position.InFrontOf(3.0f), "Test Portal", PortalType.Purple));
         }
+
+        /// <summary>
+        /// Debug command to saves the character from in-game.
+        /// </summary>
+        /// <remarks>Added a quick way to invoke the character save routine.</remarks>
+        /// <param name="session"></param>
+        /// <param name="parameters"></param>
+        [CommandHandler("save", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld)]
+        public static void Save(Session session, params string[] parameters)

```

Comment:
Useful for invoking a database save; I used this instead of logging out or waiting 5 minutes.